### PR TITLE
Update Albania holidays, add l10n support

### PR DESCRIPTION
**Pull request by KJhellico**
## Proposed change

Update Albania holidays, add l10n support (en_US, **sq**, uk).

## Type of change

- [ ] New country/market holidays support (thank you!)
- [x] Supported country/market holidays update (calendar discrepancy fix, localization)
- [ ] Existing code/documentation/test/process quality improvement (best practice, cleanup, refactoring, optimization)
- [ ] Dependency update (version deprecation/pin/upgrade)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (a code change causing existing functionality to break)
- [ ] New feature (new `holidays` functionality in general)

## Checklist

- [x] I've followed the [contributing guidelines][contributing-guidelines]
- [x] I've successfully run `make check`, all checks and tests are green

[contributing-guidelines]: https://github.com/vacanza/holidays/blob/dev/CONTRIBUTING.rst
[docs]: https://github.com/vacanza/holidays/tree/dev/docs/source

### CODE REVIEW

**Review by PPsyrius — APPROVED** [review]
LGTM 🇦🇱

File: tests/countries/test_albania.py [hunk 1]
```diff
@@ -19,69 +19,119 @@
...
             "2024-04-10",
-            # Eid al-Adha
-            "2006-01-10",
-            "2006-12-31",
-            "2015-09-23",
-            "2016-09-11",
-            "2017-09-01",
+        )
+        years_found = {dt.year for dt in self.holidays.get_named(name, lookup="startswith")}
+        self.assertTrue(set(range(1993, 2050)).issubset(years_found))
```
Comment (arkid15r) [inline tests/countries/test_albania.py:129]:
```suggestion
        self.assertTrue(
            set(range(1993, 2050)).issubset(
                {dt.year for dt in self.holidays.get_named(name, lookup="startswith")}
            )
        )
```

File: tests/countries/test_albania.py [hunk 2]
```diff
@@ -90,82 +140,81 @@ def test_islamic_holidays(self):
...
-
-        name = "Mother Teresa Canonization Day"
-        self.assertNoHolidayName(name, range(1990, 2018))
-        self.assertHolidayName(name, (f"{year}-09-05" for year in range(2018, 2050)))
+        self.assertHoliday(
+            "2006-01-10",
+            "2006-12-31",
+        )
+        years_found = {dt.year for dt in self.holidays.get_named(name, lookup="startswith")}
+        self.assertTrue(set(range(1993, 2050)).issubset(years_found))
```
Comment (arkid15r) [inline tests/countries/test_albania.py:148]:
```suggestion
        self.assertTrue(
            set(range(1993, 2050)).issubset(
                {dt.year for dt in self.holidays.get_named(name, lookup="startswith")}
            )
        )
```

**Review by arkid15r — APPROVED** [review]
LGTM

A couple of optional suggestions:

**Pull request merged**